configure.ac

```diff
@@ -56,6 +67,48 @@ AS_CASE([${RKT_STAGE1_USR_FROM}],
 
 AC_SUBST(RKT_STAGE1_USR_FROM)
 
+m4_define(DEFAULT_STAGE1_NAME,coreos.com/rkt/stage1)
```

Comment:
Just started to wonder if we shouldn't name stage1 after flavors. Something like `coreos.com/rkt/stage1-coreos`?